NAMESPACE

```diff
@@ -24,13 +24,13 @@ export(create_and_normalize_SE)
 export(data_model)
 export(fit_SE)
 export(fit_SE.combinations)
+export(gdr_matches)
 export(get_default_nested_identifiers)
 export(identify_data_type)
 export(identify_keys)
 export(map_conc_to_standardized_conc)
 export(map_df)
 export(map_ids_to_fits)
-export(matches)
```

Comment:
@bczech it seems that `pkgdown::build_site` does not handle properly exported function named `matches`.  I thought it would be faster to change it on our side. (We can also consider adding an issue in the pkdown repo).

Details:
```
> pkgdown::build_reference_index()
Called from: select_topics(match_strings, topics)
Browse[1]> c
Error in `map2()`:
ℹ In index: 1.
Caused by error in `purrr::map()` at pkgdown/R/topics.R:9:3:
ℹ In index: 24.
Caused by error:
! In _pkgdown.yml, topic must be a known topic name or alias, not "`matches`".
Run `rlang::last_trace()` to see where the error occurred.
```
```
Browse[1]> match_strings
# see 24th element
 [1] "`add_CellLine_annotation`"             "`add_Drug_annotation`"                 "`calculate_GR_value`"                  "`calculate_excess`"                   
 [5] "`calculate_matrix_metric`"             "`cleanup_metadata`"                    "`convert_mae_to_raw_data`"             "`convert_se_to_raw_data`"             
 [9] "`data_model`"                          "`data_model.character`"                "`data_model.data.table`"               "`define_matrix_grid_positions`"       
[13] "`.map_references`"                     "`.standardize_conc`"                   "`fit_SE.combinations`"                 "`get_assays_per_pipeline_step`"       
[17] "`get_default_nested_identifiers`"      "`identify_data_type`"                  "`identify_keys`"                       "`map_conc_to_standardized_conc`"      
[21] "`map_df`"                              "`map_ids_to_fits`"                     "`map_untreated`"                       "`matches`"                            
[25] "`merge_data`"                          "`order_result_df`"                     "`prepare_input.MultiAssayExperiment`"  "`prepare_input`"                      
[29] "`prepare_input.data.table`"            "`remove_drug_batch`"                   "`replace_conc_with_standardized_conc`" "`round_concentration`"                
[33] "`average_SE`"                          "`split_raw_data`"                      "`test_synthetic_data`"                
```

Reply:
Is there any other option for handling it? This function is directly copied from `grr` package so I think we shouldn't rename this function, especially by adding `gdr` to the name indicating it's our property: https://github.com/cran/grr/blob/master/R/grr.R

What about `compare` name?

Reply:
I can confirm it's the issue on `pkgdown` side. I was able to reproduce the issue with the latest `testthat` package.
I had to add `matches` to `_pkgdown.yml`:
```
gladkia@gladkia-UHMD6M testthat % git diff
diff --git a/_pkgdown.yml b/_pkgdown.yml
index 209eea3d..62abeb34 100644
--- a/_pkgdown.yml
+++ b/_pkgdown.yml
@@ -53,6 +53,7 @@ reference:
   - skip
   - teardown_env
   - set_state_inspector
+  - matches
 
 - title: Run tests
   contents:
```
And the `pkgdown::build_reference` failed in the analogous way:
```
> pkgdown::init_site()
> pkgdown::build_reference()
-- Building function reference -------------------------------------------------------------------------------------------------------------------------------------
Error in `map2()`:
ℹ In index: 6.
Caused by error in `purrr::map()`:
ℹ In index: 5.
Caused by error:
! In '_pkgdown.yml', topic must be a known topic name or alias
✖ Not 'matches'
Run `rlang::last_trace()` to see where the error occurred.
```

Reply:
I've added new issue: https://github.com/r-lib/pkgdown/issues/2396.

Reply:
@bczech I've updated to `grr_matches` in 24ea3a25902feed638f7db097e7651b0b608149b.